snapshots/war/latest/ucdp_ced_v25_01_25_09.csv.dvc

```diff
@@ -0,0 +1,28 @@
+meta:
+  origin:
+    producer: Uppsala Conflict Data Program
+    title: Candidate Events Dataset
+    description: |-
+      The UCDP Candidate Events Dataset (UCDP Candidate) is based on UCDP Georeferenced Event Dataset (UCDP GED), but published at a monthly release cycle. It makes available monthly releases of candidate events data with not more than a month's lag globally. See codebook for similarities and differences between the two products.
+
+      You can find more notes at https://ucdp.uu.se/downloads/candidateged/ucdp-candidate-codebook1.4.pdf
```

Comment:
I thought it's not obvious what do you mean by codebook. You could say:
```

See [codebook](https://ucdp.uu.se/downloads/candidateged/ucdp-candidate-codebook1.4.pdf) for similarities and differences between the two products, as well as other relevant notes.
```
And then remove the next sentence.